### PR TITLE
Add vendor_revision property for sff-Parser

### PR DESCRIPTION
**Pull request by karthikeyan-arrcus**
Parsed and populated the vendor_rev field

### CODE REVIEW

**Comment by karthikeyan-arrcus** [comment]
@sonoble Kindly look onto it and merge it ASAP

**Comment by karthikeyan-arrcus** [comment]
Hii @sonoble, can u review and merge this commit

**Comment by karthikeyan-arrcus** [comment]
Any update on this @sonoble?

**Review by sonoble — APPROVED** [review]

File: modules/sff/module/src/sff.c [hunk 1]
```diff
@@ -1013,6 +1024,7 @@ sff_info_to_json(sff_info_t* info, cJSON** cjp)
     cjson_util_add_string_to_object(cj, "vendor", "%-16.16s", info->vendor);
     cjson_util_add_string_to_object(cj, "model", "%-16.16s", info->model);
     cjson_util_add_string_to_object(cj, "serial", "%-16.16s", info->serial);
+    cjson_util_add_string_to_object(cj, "serial", "%-4.4s", info->vendor_rev);
```
Comment (KanjiMonster) [inline modules/sff/module/src/sff.c:1027]: You are adding a second "serial", this probably was supposed to be "vendor-revision" or so.

**Pull request merged**